sound/soc/sof/sof-priv.h

```diff
@@ -56,6 +56,9 @@
 #define SOF_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE | \
 	SNDRV_PCM_FMTBIT_S32_LE | SNDRV_PCM_FMTBIT_FLOAT)
 
+/* Set upper limit for TLV bytes length to 1 MB */
+#define SND_SOF_CTL_TLV_LENGTH_MAX 1048576
```

Comment:
(1024*1024) or something easier to read.

Reply:
ok

Reply:
Done.